docs/glossary.md

```diff
@@ -302,10 +320,73 @@ _Tags: [term](/tags/term), [explanation](/tags/explanation)_
 
 _Tags: [term](/tags/term), [explanation](/tags/explanation)_
 
+#### [Nexus API Definitions](/nexus#nexus-api-defintions)
+
+A common code package, schema, or documentation that a Caller can use to obtain Service and Operation names as associated input/output types a Service will accept for a given Operation.
+
+_Tags: [term](/tags/term), [nexus](/tags/nexus), [explanation](/tags/explanation)_
+
+#### [Nexus API Registry](/nexus#nexus-api-registry)
+
+An account-scoped list of Endpoints and Services available for use within a Temporal Cloud account, including an Endpoint description of the Services and Operations which the Endpoint supports. 
+
+_Tags: [term](/tags/term), [nexus](/tags/nexus), [explanation](/tags/explanation)_
+
+#### [Nexus Client](/nexus#nexus-client)
+
+Used by the Caller to connect to a Nexus Endpoint and Nexus Service.
+
+_Tags: [term](/tags/term), [nexus](/tags/nexus), [explanation](/tags/explanation)_
+
+#### [Nexus Endpoint](/nexus#nexus-endpoint)
+
+A Nexus Endpoint is like an API proxy that can serve one or more Nexus Services from a target such as a Worker listening on a target Temporal Namespace and target Task Queue. 
+This allows service providers to present a clean API contract and hide the underlying implementation, which may consist of many internal Workflows. 
+Multiple Nexus Endpoints can target the same Namespace, and over time a Nexus Endpoint will be able to span multiple Namespaces with service routing rules.
+
+_Tags: [term](/tags/term), [nexus](/tags/nexus), [explanation](/tags/explanation)_
+
+#### [Nexus Operation](/nexus#nexus-operation)
+
+An arbitrary-length service call that may be synchronous or asynchronous, short-lived, or long-lived, used to invoke operations across Namespace, Temporal Service, and cell boundaries and into arbitrary external systems.
+
+_Tags: [term](/tags/term), [nexus](/tags/nexus), [explanation](/tags/explanation)_
+
+#### [Nexus Operation Events](/nexus#nexus-operation-events)
+
+History events that surface in the Caller Workflow to indicate the state of an Operation including `Op Scheduled`, `Op Started`, `Op Completed`.
+
+_Tags: [term](/tags/term), [nexus](/tags/nexus), [explanation](/tags/explanation)_
+
+#### [Nexus Operation Handle](/nexus#nexus-operation-handle)
+
+Returned to the Workflow that started an operation to interact with the operation, for example `handle.WaitStarted()`, `handle.GetResult()`.
+
+_Tags: [term](/tags/term), [nexus](/tags/nexus), [explanation](/tags/explanation)_
+
+#### [Nexus Operation Handler](/nexus#nexus-operation-handler)
+
+The handler code in a Handler Worker, used to process service calls for a given Operation. Operation Handlers often run in the same Worker as the Workflows they abstract, to provide a service interface that can be used across Namespace boundaries.
+
+_Tags: [term](/tags/term), [nexus](/tags/nexus), [explanation](/tags/explanation)_
+
+#### [Nexus Operation Reference](/nexus#nexus-operation-reference)
```

Comment:
outdated

```suggestion
```

Reply:
It's not outdated, it's just a Go specific concept.